src/main/kotlin/telegram/MessageMarkedUpContext.kt

```diff
@@ -0,0 +1,13 @@
+package org.example.telegram
+
+data class MessageMarkedUpContext(
```

Comment:
It's Kotlin, there is no one class per file rule
Keep things together if they are always being used together

Reply:
Name too long and does not help with understanding of what this class does
`MessageCallbackContext` is better or even just `MessageCallback` and rename function to `then` or something around those lines

That's if we only have this class, but since we also have `MessageSentContext` and you copy-pasted `callback` function from one of them to another completely without any changes, inheritance makes sense here. `MessageSentContext` is just an extension of this class.
Don't forget `sealed`.
In this case it makes sense to still call this one `MessageCallback` and another one something like `MessageKeyboardContext`, because keyboard is technically a callback, since we create it after we send the message